src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -513,7 +513,7 @@ public T[] ToArray()
         {
             // Short path if the bag is empty
             if (_headList == null)
-                return new T[0];
+				return Array.Empty<T>();
```

Comment:
On the flip side. If this change is taken then a test of `Assert.Same(new ConcurrentBag<string>().ToArray(), ConcurrentBag<string>().ToArray())` would mean any change back would at least have to justify why they removed they performance gain by justifying removing the now-breaking test.